This patch disables the submodule check that occurs during the configuration 
step. This is necessary when building from a source tarball, as opposed to a 
git clone, and as required to include a checksums for easybuild.  
author: Luke Labrie-Cleary
diff --unified --recursive --text openmc-0.15.0.orig/CMakeLists.txt openmc-0.15.0.new/CMakeLists.txt
--- openmc-0.15.0.orig/CMakeLists.txt	2024-06-22 03:28:56.000000000 +0200
+++ openmc-0.15.0.new/CMakeLists.txt	2024-07-16 13:50:18.535439839 +0200
@@ -237,12 +237,6 @@
   endif()
 endif()
 
-# Check to see if submodules exist (by checking one)
-if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/vendor/pugixml/CMakeLists.txt")
-  message(FATAL_ERROR "The git submodules were not downloaded! GIT_SUBMODULE was \
-    turned off or failed. Please update submodules and try again.")
-endif()
-
 #===============================================================================
 # pugixml library
 #===============================================================================
